test/wveOLAS.js

```diff
@@ -48,8 +48,9 @@ describe("Wrapped Voting Escrow OLAS", function () {
             result = await wve.getVotes(AddressZero);
             expect(result).to.equal(0);
 
-            result = await wve.getPastTotalSupply(0);
-            expect(result).to.equal(0);
+            await expect(
+                wve.getPastTotalSupply(0)
+            ).to.be.reverted;
```

Comment:
As we rolled back to the original veCRV behavior, the provided block number for `getPastTotalSupply()` can't be lower than the zero supply point block number, or the block number of the contract deployment, as outlined in the `Vulnerabilities_list` file.